v2/index_gen.go

```diff
@@ -76,7 +80,13 @@ func LoadIndex(idx index.Index, v1r io.Reader, opts ...Option) error {
 		if err != nil {
 			return err
 		}
-		records = append(records, index.Record{Cid: c, Offset: uint64(sectionOffset)})
+
+		if c.Prefix().MhType != multihash.IDENTITY || o.IncludeIdentityCIDs {
+			if uint64(cidLen) > o.MaxIndexCidSize {
```

Comment:
coverage of this case might be nice - it could even fit in one of the existing tests here without needing a special test just for it

Reply:
There is a test for large CIDs already in this PR [here](https://github.com/ipld/go-car/blob/9f9f687b3606dae462007d93cc0e8684129435f5/v2/blockstore/readwrite_test.go#L778-L795). Do you think it's missing something?

Reply:
yeah, it's not using the default (common) path but setting an unrealistic max to break, that was my thought
but this is a _nice to have_, not a demand for busy-work if you're comfortable with this as is

Reply:
Thanks for pointing this out; captured https://github.com/ipld/go-car/issues/241